client/src/components/container/user/EventPage/index.js

```diff
@@ -1,27 +1,50 @@
 import React, { Component } from 'react'
-import { Button } from "antd";
+import { Button ,Alert} from "antd";
 
 import EventPageContent from '../EventPageContent/EventPageContent'
 import PopupBtnEventcomp from '../PopupBtnEventcomp/PopupBtnEventcomp'
+import Axios from 'axios';
 
 import './style.css'
 
 class EventPage extends Component {
-
-  render() {
+  state={
+    image:"",
+    title:"",
+    otherEventProps:[],
+    error:""
+  }
+  componentDidMount(){
     const { location: { state: {info : { image, ...otherEventProps }} } } = this.props;
     const { location: { state: {info:{title}} } } = this.props;
+    const { match: { params: { eventCode } } } = this.props;
+    if(this.props.location.state.info){
+      this.setState({image,title,otherEventProps})
```

Comment:
If the state of props is empty , you will got an error here , info isn't defined so fix checking as the following : 
```suggestion
const {location:{state} } = this.props;
    if(state){
       const {info : { image, ...otherEventProps } }  = state;
       const {info:{title} } = state;
      this.setState({image,title,otherEventProps})
```